cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Broker.java

```diff
@@ -583,6 +589,46 @@ void setReplicaLoad(TopicPartition tp, AggregatedMetricValues aggregatedMetricVa
     return brokerMap;
   }
 
+  /**
+   * Get detailed disk utilization information of the broker.
+   *
+   * @return The detailed disk utilization information.
+   */
+  public Map<String, Double> diskUtils() {
+    if (_diskByLogdir == null || _diskByLogdir.isEmpty()) {
+      return null;
+    }
+    // If information of replica placement over disk is not populated, return nothing.
+    if (_diskByLogdir.values().stream().mapToInt(disk -> disk.replicas().size()).sum() == 0) {
+      return null;
+    }
+    Map<String, Double> diskUtilMap = new HashMap<>(_diskByLogdir.size());
+    for (Map.Entry<String, Disk> entry : _diskByLogdir.entrySet()) {
+      diskUtilMap.put(entry.getKey(), entry.getValue().load());
+    }
+    return diskUtilMap;
+  }
+
+  /**
+   * Get detailed disk capacity information of the broker.
+   *
+   * @return The detailed disk capacity information.
+   */
+  public Map<String, Double> diskCapacities() {
+    if (_diskByLogdir == null || _diskByLogdir.isEmpty()) {
+      return null;
+    }
+    // If information of replica placement over disk is not populated, return nothing.
+    if (_diskByLogdir.values().stream().mapToInt(disk -> disk.replicas().size()).sum() == 0) {
+      return null;
```

Comment:
the capacity information of disk comes from [BrokerCapacityConfigResolver](https://github.com/linkedin/cruise-control/blob/ceed1c2ba7862360ab9a5f6f5c76ff6373f65aa3/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java#L512) and the replica placement information comes from [describeLogdirRequest](https://github.com/linkedin/cruise-control/blob/ceed1c2ba7862360ab9a5f6f5c76ff6373f65aa3/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java#L534). 

So they are kind unrelated, you can have disk capacity information, but no replica placement information(i.e disk.replicas().size()==0) because 
1) we are create a cluster model with [populateDiskInfo](https://github.com/linkedin/cruise-control/blob/ceed1c2ba7862360ab9a5f6f5c76ff6373f65aa3/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java#L469) set to false 
2) describeLogdirRequest get error response from certain broker.